py/desispec/quicklook/merger.py

```diff
@@ -0,0 +1,45 @@
+#
+# A class to merge quicklook qa outputs.
+#  
```

Comment:
If this is intended to be used for offline (soon)
as I believe it should, then I strongly request it
be written as a stand-alone bit in qa.utils
and then you can wrap it within a QL object in
quicklook to your heart's desire.

Thanks.

Reply:
@profxj  Could you please clarify this comment? This is a standalone class, it doesn't have any dependencies other than yaml, desiutil.io.yamlify and if you want to get json output as well, json module. It is imported into a project by 
from desispec.quicklook.merger import QL_QAMerger

What do you mean standalone? Do you mean you don't want to use line above?

Regards,
Sami

Reply:
I mean it has QL written all over it, @samikama 

That at least implies it is to be used for QL,
not offline QA.